product-features/self-service-listing.md

```diff
@@ -1,122 +1,176 @@
+* Check your address activity on the wallet.
+{% endhint %}
+
+</details>
+
+### Step 2: Contract creation
+
+Once the transaction from Step 1 is completed, you will see the checkbox labeled `Deposit Anchor Token ✅` marked as done. The ALEX team will review the submitted information and create a specific contract (a wrapped version) for your token to interact with the AMM DEX. This process may take between 24 and 48 hours.
+
+### Step 3: Deposit listing token
+
+Once the `Contract ready ✅` checkbox is marked as done, you're ready to deposit the listing token balance. This step involves interacting with a smart contract, so be sure to review the transaction details, paying particular attention to the amount to transfer. By accepting this transaction, you agree to transfer the initial liquidity of the listing token from your wallet to the ALEX smart contract.
+
+### Step 4: Pool creation success
+
+Once the `Deposit Listing Token ✅` transaction is completed and the `Open pool ✅` checkbox is marked as done, your pool will be automatically ready for use. The new pool will appear as an ALEX Pool under the `Self Listed` tab on [app.alexlab.co/pool](https://app.alexlab.co/pool).
+
+🤝 After completing this step, you (and everyone) can start trading the token pair on ALEX DEX 🤝
+
+<figure><img src="../.gitbook/assets/self-service-listing/pool-creation-successful.jpg" alt="Pool creation successful" width=""><figcaption><p>Pool creation successful.</p></figcaption></figure>
+
+{% hint style="warning" %}
+If you have added a custom `start-block` configuration, the pool will be unavailabe until that block is reached.
+{% endhint %}
+
+### Step 5: Provide additional token information (optional)
+
+To make your token visible on the ALEX Token List at [app.alexlab.co/token-list](https://app.alexlab.co/token-list), provide additional token information. Click on `Customer Support` on the [Self-Service Listing page](https://app.alexlab.co/self-service-listing) or contact us via Telegram at [t.me/ALEXselfservice ](https://t.me/ALEXselfservice) to submit the information (e.g. X accont, Discord, official website).
+
+<figure><img src="../.gitbook/assets/self-service-listing/token-list.png" alt="Token List example" width=""><figcaption><p>Token List example.</p></figcaption></figure>
+
+ALEX requires a [Coingecko](https://www.coingecko.com/) or [CoinMarketCap](https://coinmarketcap.com/) token listing to verify the provided social media information before uploading it to the official list at [app.alexlab.co/token-list](https://app.alexlab.co/token-list).
+
+Thanks for creating your pool on the ALEX DEX 🎉 📈 
+
+<!-- 
+Summarized Steps:
+
+1) User submits token information, balances and config params. Within this same transaction, transfers the anchor token balance.
+
+2) User waits for token confirmation from ALEX side (this involves deploying the wrapped version, but the user doesn't get to know this detail).
```

Comment:
is this visible to users? if so it doesn't make sense to include "but the user doesn't get to know this detail"

Reply:
Not visible to users, as they are markdown comments. They are only visible in the git repository. In this case, it is just a steps summarize. However, we removed `(this involves deploying the wrapped version, but the user doesn't get to know this detail)`.